charts/k8ssandra/values.yaml

```diff
@@ -169,53 +198,148 @@ stargate:
+        # tls:
+        #   client_auth:
+        #     # -- Collection of Secrets containing TLS certificates. KEYS ARE NOT REQUIRED HERE
+        #     ca_secrets:
+        #       - ca_secret_1
+        #   domains:
+        #     - example.com
+        #   # -- Secret containing the TLS certificate and key for the listener
+        #   secretName: tls-secret
+
+reaper:
+  # -- When enabled, Reaper automatically sets up repair schedules for all
+  # non-system keypsaces. Repear monitors the cluster so that as keyspaces are
+  # added or removed repair schedules will be added or removed respectively.
+  autoschedule: false
+
+  # -- Enable Reaper resources as part of this release. Note that Reaper uses
+  # Cassandra's JMX APIs to perform repairs. When Reaper is enabled, Cassandra
+  # will also be configured to allow remote JMX access. JMX authentication
+  # will be configured in Cassandra with credentials only created for Reaper
+  # in order to limit access.
+  enabled: true
+
+  # Configures the Reaper container image to use.
+  image:
+    # -- Specifies the container repository for cassandra-reaper
+    repository: docker.io/thelastpickle/cassandra-reaper
+    # -- Tag of an image within the specified repository
+    tag: 2.1.3
+
+  # -- Configures the Cassandra user used by Reaper when authentication is
+  # enabled. If neither cassandraUser.secret nor casandraUser.username are
+  # set, then a Cassandra user and a secret with the user's credentials will
+  # be created. The username will be reaper. The secret name will be of the
+  # form {clusterName}-reaper. The password will be a random 20 character
+  # password. If cassandraUser.secret is set, then the Cassandra user will be
+  # created from the contents of the secret. If cassandraUser.secret is not
+  # set and if cassandraUser.username is set, a secret will be generated using
+  # the specified username. The password will be generated as previously
+  # described.
+  cassandraUser:
+    secret: ""
+    username: ""
+
+  # -- Configures JMX access to the Cassandra cluster. Reaper requires remote
+  # JMX access to perform repairs. The Cassandra cluster will be configured
+  # with remote JMX access enabled when Reaper is deployed. The JMX access
+  # will be configured to use authentication.
+  jmx:
+    # -- Username that Reaper will use for JMX access. If left blank a random,
+    # alphanumeric string will be generated.
+    username: ""
+
+    # -- Password that Reaper will use for JMX access. If left blank a random,
+    # alphanumeric string will be generated.
+    password: ""
+
+  ingress:
+    # -- Enables Reaper ingress definitions.
+    enabled: false
+
+    # -- Hostname to use for routing requests to the repair UI. If
+    # using a local deployment consider leveraging dynamic DNS services like
+    # xip.io. Example: `repair.127.0.0.1.xip.io` will return `127.0.0.1` for
+    # DNS requests routing requests to your local machine.
+    host:
```

Comment:
The comments here should indicate that `host` is required when reaper ingress is enabled.

Reply:
Done.